referencia/webpay/README.md

```diff
@@ -1372,7 +1372,7 @@ result.tbkUser;
 
 Nombre  <br> <i> tipo </i> | Descripción
 ------   | -----------
-responseCode  <br> <i> xs:int </i> | Código de retorno del proceso de inscripción, donde 0 (cero) es aprobado.
+responseCode  <br> <i> xs:int </i> | Código de retorno del proceso de inscripción, donde 0 (cero) es aprobado. <br> -96: Se llama el método sin realizar previamente la autenticación bancaria.
```

Comment:
I still think this should be explained a bit better. For example: "-96: Se invocó `finishInscription` sin que el tarjetahabiente se haya autenticado correctamente con su banco en el flujo Webpay" (some people might not understand right away that "autenticación bancaria" refers to that step; they can mix it up with "autorización bancaria", or even not understand it at all)